examples/python/printing-client/printing-client.py

```diff
@@ -0,0 +1,84 @@
+# Copyright 2018 Infostellar, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""The Python implementation of the gRPC stellarstation-api client."""
+
+import base64
```

Comment:
Did you / can you run this file through `yapf` for formatting?

Reply:
The code formatted by yapf with 
`yapf --style='{based_on_style: google, indent_width: 2}' printing-client.py `

Reply:
Ah - let's use pep8 style for our python code, we use Google style on Java since there's no official style there but Python does have an official style.